src/app/mini-me/mini-me.page.html

```diff
@@ -5,5 +5,8 @@
 </ion-header>
 
 <ion-content>
+  <ion-text color="primary">
```

Comment:
Looks like this is conflicting with Dar's recent work. Since you don't need this particular change, Try deleting your `<ion-content>` and replacing it with

```
<ion-content>
  <swiper [config]="config">

    <ng-template swiperSlide>
      <app-mini-me-setup></app-mini-me-setup>
    </ng-template>

    <ng-template swiperSlide>
      <app-mini-me-builder></app-mini-me-builder>
    </ng-template>

    <ng-template swiperSlide>
      <app-mini-me-confirm></app-mini-me-confirm>
    </ng-template>

  </swiper>

</ion-content>
```

Then try commit and push again, and let's see if the conflict will go away